p4proto/kctrl/switchapi/switch_pd_utils.h

```diff
@@ -50,17 +50,24 @@ bf_status_t switch_pd_allocate_handle_session(const bf_dev_id_t device_id,
                                               bf_rt_info_hdl **bfrt_info_hdl_t,
                                               bf_rt_session_hdl **session_t);
```

Comment:
We can remove this one as well right? Also, can you rebase the branch and take care of DCO check?

Reply:
Ack